pyspectral/rsr_reader.py

```diff
@@ -130,11 +130,17 @@ def _get_filename(self):
 
         LOG.debug('Filename: %s', str(self.filename))
         if not os.path.exists(self.filename) or not os.path.isfile(self.filename):
-            # Try download from the internet!
             LOG.warning("No rsr file %s on disk", self.filename)
```

Comment:
Should this warning be moved to the `else:` block?

Reply:
No. I don't think so. If the file is not there on disk I issue a warning. Either the user hasn't been running/updating the data for a while or there is no support for this platform/sensor.

Reply:
Ah ok. I see now. Now I understand. This issue was caused by unsupported sensors and it trying to download the LUTs every time trying to find the LUT for the unsupported data.

Reply:
That could have been the thing. But, in case of Ralph's example, it think only explains the double download, but it doesn't explain why it kept downloading whenever he ran the script again. Sice Himawari is/was supported. So, maybe it is related to how data are stored when using conda installations?